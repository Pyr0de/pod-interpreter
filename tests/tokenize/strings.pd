"simple"
"space1 space2 space3"
"quotes '' ' "
'single quotes'
"fails
'fails again
