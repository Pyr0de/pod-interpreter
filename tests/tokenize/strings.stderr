ExitCode: 65
[line 5] Unterminated String: "fails
[line 6] Unterminated String: 'fails again
Tokenize Error
